syntax = "proto3";

package echo;

service EchoService {
  rpc getAllMetrics (getAllMetricsParams) returns (AllMetrics) {}
  rpc getMetadataForMetrics (getMetadataForMetricsParams) returns (AllMetrics) {}
  rpc getHistoryForMetrics (getMetadataForMetricsParams) returns (AllMetrics) {}
}

message getMetadataForMetricsParams {
  string idMetric = 1;
}

message getAllMetricsParams {}

message AllMetrics {
  string result = 1;
}